core/src/text/fontContext.cpp

```diff
@@ -185,6 +185,11 @@ bool FontContext::layoutText(TextStyle::Parameters& _params, const std::string&
                      (metrics.aabb.y + height * 0.5) * TextVertex::position_scale);
 
     auto it = _quads.begin() + quadsStart;
+    if (it == _quads.end()) {
```

Comment:
Missing glyphs will just be ignored - i.e. not added to the LineLayout and not adding 'empty space' to it